bedrock/firefox/templates/firefox/welcome/page1.html

```diff
@@ -86,6 +89,18 @@ <h3 class="c-picto-block-title">{{ _('Stay in the know') }}</h3>
   {% endif %}
 {% endblock %}
 
+{% block secondary_cta %}
+  <p>
+      {{ monitor_button(
+        entrypoint=_entrypoint,
+        utm_campaign=_utm_campaign,
+        button_text=_('Check Your Breach Report'),
+        cta_type=_cta_type,
+        cta_position='secondary'
```

Comment:
Should this also have `utm_content='secondary'` to match the button metrics above?

Reply:
nice catch! some leftover code. 
I was in a discussion about whether `utm_content` should be passed to monitor to share info about which button was clicked. However, I don't think we need to pass this info because we are already collecting whether "primary" or "secondary" was clicked from `data-cta-position` on our end.

Reply:
I am going to change both to `cta_position`

Reply:
If we don't need to pass those metrics over in the flow request, then we should remove the code that does it? https://github.com/mozilla/bedrock/blob/master/media/js/base/mozilla-monitor-button.js#L45-L56

I guess my question is, what's required and what's not?

Reply:
We should keep that code because `utm_content` is optional.
> `utm_content`: Declared when more than one piece of content (on a page or at a URL) links to the same place, to distinguish between them.

the parameters that are required are:
- `entrypoint`
- `utm_campaign`

according to https://bedrock.readthedocs.io/en/latest/firefox-accounts.html#linking-to-monitorfirefoxcom

(though, I'm pretty sure `form_type='button'` and `'utm_medium=referral'` is also required.)

I guess the question here is that is `utm_content=primary/secondary` helpful information for monitor to have? I will confirm.

Reply:
Ok I'm confused - the code I'm linking to above only passes over those values in the flow request. Removing that code will still keep those values in the final href. Unless I'm not following what you mean?

Reply:
oh! I see what you are saying. "Why are we passing these values specifically to the flow request if they are not required?" a good question! I think this is copy pasta from `mozilla-fxa-form.js`?